distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -196,74 +217,104 @@ If a self intersecting polygon is used to perform a geographic search, the polyg
 image::convex-hull-transform-example.png[]
 ====
 
-==== Results Tab
+===== Refining Search Results
 
-Returned search results can be refined further, bookmarked, and/or downloaded from the *Results* tab.
+Returned search results can be refined further, bookmarked, and/or downloaded from the *Search* tab.
 Result sets are color-coded by source as a visual aid.
 There is no semantic meaning to the colors assigned.
 
-.Results Tab Options
-image::results-tab-options.png[results tab options]
+.Search Results Options
+image::search-results-options.png[search results options]
 
-. On the *Results* tab, select a search fr4om the drop-down list.
-. Perform any of these action upon the results list of the selected query:
-.. Filter the result set locally (does not re-execute the search),
-.. Customize results sorting (Default: Title in Ascending Order).
+. On the *Search* tab, select a search from the drop-down list.
+. Perform any of these actions on the results list of the selected search:
+.. Filter the result set locally. This does not re-execute the search.
+.. Customize results sorting. The default sort is by title in ascending order.
 .. Toggle results view between *List* and *Gallery*.
 
+===== Search Result Options
+
+.Options for each individual search result
+* *Download*: Downloads the result's associated product directly to the local machine.. This option is only available for results that have products.
```

Comment:
✏️  looks like there is an extra `.` after `machine`

Reply:
fixed